### PR TITLE
UI-6616 - Update the documentation

### PR DESCRIPTION
**Pull request by antoinetissier**
📜

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,30 +22,24 @@ npm install activeui-migration
...
 
 ![](/documentation/download-content.gif)
 
-### Optional: Download the content of the pivot folder
-
-:warning: If there are calculated measures to be migrated from ActiveUI-4 to the ActiveUI-5 content servers, this file is required. If this is not the case, this file can be disregarded.
-
-Right-click the folder named **pivot** and click **Export**. This will download a file called _content of pivot.json_. Move this file into your `migration` folder.
+:warning: Importing/exporting at the root of the file tree is available in atoti+ Admin UI version 5.1.0 and higher.
+If you don't see these actions in the context menu, it means that you are using an older Admin UI version.
```
Comment (NZepeda) [inline README.md:41]:
```suggestion
If you don't see these actions in the context menu, it means that you are using an older Admin UI version and will have to upgrade.
```
Just to make it clearer what you are referring to below when you say "you can do so...."

File: README.md [hunk 2]
```diff
@@ -66,15 +60,14 @@ Copy and paste the snippet below into this file. It is not a valid JSON object.
...
+Choose a key to identify this server and replace `"my-server"` by it. See [the ActiveUI tutorial](https://activeviam.com/activeui/documentation/5.1.0/docs/tutorial/runningAQuery#serverkey) for more details about server keys.
 
-Download the data model from your ActivePivot server. The URL of the relevant service ends with _"discovery"_. You will find its full URL in the documentation under `/webservices`.
+Download the data model from your ActivePivot server. The URL of the relevant REST service ends with _"discovery"_. You can find the documentation for this REST service [here](https://artifacts.activeviam.com/documentation/rest/6.0.0/activepivot.html#cube_discovery_get).
 
 For instance, considering the ActivePivot server you used while going through the tutorial:
 
 - it is deployed at https://activeui-tutorial-server.activeviam.com:9090/pivot
-- its `webservices` documentation is accessible at https://activeui-tutorial-server.activeviam.com:9090/pivot/webservices
```
Comment (NZepeda) [inline README.md:76]: Is the `webservices` endpoint not required?

File: README.md [hunk 3]
```diff
@@ -99,41 +92,55 @@ At this stage, your `migration` folder looks like this:
...
-For example, suppose you have 200 ActiveUI 4 bookmarks in which you use Page Filters and/or Page Context Values.
-These plugins are not supported anymore in ActiveUI 5 as it was deemed a better user experience to remove them altogether.
-If you want to follow this UX and remove these widgets programmatically, you can run the migration CLI with the extra option:
+```
+npx migrate --help
+```
+
+In particular:
+
+- The `--debug` and `--stack` allow respectively to generate an error report and the stack of those errors. This is useful to understand and address any error that may occur the migration.
```
Comment (NZepeda) [inline README.md:120]:
```suggestion
- The `--debug` and `--stack` respectively allow for an error report generation and the stack of those errors. This is useful to understand and address any error that may occur the migration.
```
Little bit of a rewording, what do you think?

**Review by NZepeda — CHANGES_REQUESTED** [review]
Got a few suggestions but looking good!

**Review by NZepeda — APPROVED** [review]
Approving but I had one minor suggestion before you merge 👍

**Comment by antoinetissier** [comment]
@eliasinho can you add the things that you spotted ?

**Pull request merged**